lib/racecar/runner.rb

```diff
@@ -130,11 +130,6 @@ def process_method
 
     def consumer
       @consumer ||= begin
-        # Manually store offset after messages have been processed successfully
-        # to avoid marking failed messages as committed. The call just updates
-        # a value within librdkafka and is asynchronously written to proper
-        # storage through auto commits.
-        config.consumer << "enable.auto.offset.store=false"
```

Comment:
This was causing a constantly growing array when running multiple integration specs outside of this repo with my Racecar consumer. Moving it to [`ConsumerSet`](https://github.com/zendesk/racecar/blob/2ef014c5f2488d1d5af3e5d434587bfd6a7bd0d2/lib/racecar/consumer_set.rb#L218C9-L218C9)  with the other consumer defaults also ensures that `Racecar.config.consumer` will contain  only user provided config options similar to the `Racecar.config.producer` array.